enterprise/internal/batches/resolvers/credential_test.go

```diff
@@ -60,3 +63,13 @@ func TestUnmarshalBatchChangesCredentialID(t *testing.T) {
 		}
 	}
 }
+
+func TestCommentSSHKey(t *testing.T) {
+	publicKey := "public"
+	sshKey := commentSSHKey(&auth.BasicAuthWithSSH{BasicAuth: auth.BasicAuth{Username: "foo", Password: "bar"}, PrivateKey: "private", PublicKey: publicKey, Passphrase: "pass"})
+	expectedKey := publicKey + " Sourcegraph " + globals.ExternalURL().Host
+
+	if have, want := sshKey, expectedKey; have != want {
+		t.Errorf("found wrong ssh key: want=\"%+v\", have=\"%+v\"", publicKey, expectedKey)
```

Comment:
Since both values are just a string here, you can use `%q` for quoted string :) 
```suggestion
		t.Errorf("found wrong ssh key: want=%q, have=%q", publicKey, expectedKey)
```

Reply:
🤯 Thank you!!